### DISCUSSION TITLE
Offers report

### DISCUSSION THREAD

**Discussion by joshbis** [cwendt94/espn-api#244] [ESPN API]
Anyone know whether it's possible to pull the Free Agent Offers Report that shows everyone who put in a claim on a player regardless of successful claims? I was going to see which players got the most bids and how big of a spread there was per player on FAAB spending.

**Comment by cwendt94** [comment]
I am not in a FABB league so its hard to figure out which API's ESPN have exposed for that data but someone did put in the initial work to get the FABB data https://github.com/cwendt94/espn-api/pull/59. This could be a good starting point to test with your league.

**Comment by joshbis** [comment]
Thanks for this pointer. It is very weird that ESPN posts this data through a seemingly different stream -- it's under "Players" whereas recent activity is under "League". 

It also shows up in non-FAAB leagues, just without the bid offers (i.e., it shows all submitted claims and whether they were successful or not); so my guess is that these rely on the same underlying infrastructure.
